user_management/ui/views.py

```diff
@@ -35,4 +41,12 @@ def dispatch(self, request, *args, **kwargs):
     def get(self, request, *args, **kwargs):
         self.activate_user()
         messages.success(request, self.success_message)
+
+        if getattr(settings, 'USER_MANAGEMENT_AUTO_LOGIN', False):
```

Comment:
It's going to be worth considering grouping the user management settings under a single group soon. (ie `settings.USER_MANAGEMENT['LOG_IN_ON_SIGNUP']